spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/interpreter/rules/mapping/MappingRuleInterpreter.scala

```diff
@@ -26,9 +26,10 @@ import za.co.absa.enceladus.model.conformanceRule.{ConformanceRule, MappingConfo
 import za.co.absa.enceladus.model.{Dataset => ConfDataset}
 import za.co.absa.enceladus.utils.error._
 import za.co.absa.enceladus.utils.transformations.ArrayTransformations
-import za.co.absa.enceladus.utils.transformations.ArrayTransformations.arrCol
 import za.co.absa.enceladus.utils.udf.UDFNames
+import za.co.absa.standardization.udf.{UDFNames => StandardizationUDFNames}
```

Comment:
I can see that we have 2 UDFNames classes, and some of the attributes are the same. Just an idea - maybe a better solution could be to have three:
* `BaseUDFNames` class, holding attributes that are present currently in both classes, and it would have method `uniqueUDFName` as well (where to put it - maybe `spark-commons`?)
* `StandardizationUDFNames` class, holding only standardization-related attributes and maybe deriving from BaseUDFNames class
* `ConformanceUDFNames` class - similar principle as the one above

Not sure if we can split it like that at the moment, since some of it is in our library and some in Enceladus (since the conformance is only in Enceladus and not separated). 

Maybe for now, at least in line 29 we could do something like:

`import za.co.absa.enceladus.utils.udf.{UDFNames => ConformanceUDFNames}`

to indicate that this UDFNames one is meant to be used for conformance; or just rename it to `ConformationUDFNames`

Reply:
I actually had to re-add `"errorColumnAppend"`. I hope general error handling will address these. 🤷‍♂️